### PR TITLE
Project 5: Design Teardown

### PR DESCRIPTION
**Pull request by enaburekhan**
**Smashing magazine clone webpage**

In this project, we:

- Use CSS grid to layout the articles
- Use flex-box for positioning and alignment
- Use transform rotate to make the images and icons rotate

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+                    <div class="dflex article-content-sub">
+                        <div class="alignc dflex dolor">
+                            <div><i class='fas'>&#xf406;</i></div>
+                            <p>Lorem ipsum dolor</p>
+                        </div>
+                        <div>
+                            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quod, sapiente est dolores, ea
+                                eius
+                                illum sunt debitis placeat adipisci nam, laudantium maxime inventore expedita vero?
+                                <!--<span><i class='far'>&#xf27a;</i></span>-->
```
Comment (MuriloRoque) [inline index.html]: - [ ] Please, remove commented code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,696 @@
...
```
Comment (MuriloRoque) [inline index.html]: - [ ] Please, remove commented code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,696 @@
...
+                    <div class="dflex article-content-sub">
+                        <div class="dflex alignc dolor">
+                            <div><i class='fas'>&#xf406;</i></div>
+                            <p>Lorem ipsum dolor</p>
+                        </div>
+                        <div>
+                            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quod, sapiente est dolores, ea
+                                eius
+                                illum sunt debitis placeat adipisci nam, laudantium maxime inventore expedita vero?
+                                <!--<span><i class='far'>&#xf27a;</i></span>-->
```
Comment (MuriloRoque) [inline index.html]: - [ ] Please, remove commented code.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @enaburekhan and @dibdas :wave: 

### Wow, your project is awesome! I have no words to describe it, but it's definitely one of the best I saw :smile: 

### We only have two CSS bad practices to work on:

- [ ] You are using a lot of repeating properties in your CSS, I recommend you to create specific classes for them (check the frequency):

       8 |   flex-direction: column;
       8 |   justify-content: space-between;
       7 |   background-color: gray;
       7 |   border-radius: 5px;
       6 |   color: black;
       6 |   height: 8vh;
       6 |   width: 80%;

- [ ] Please check my comments in your code, there are some lines with commented pieces of code.

## Summary

- The page structure and layout is great, that's exactly how it should be done :+1:
- Your project is very professional, keep up the good work :rocket: 
- Your HTML code is good, you used semantic tags everywhere, I really liked it :smile:
- Apart from what I mentioned above, you are doing CSS good practices :smile: 

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by enaburekhan** [comment]
**Code review comments effected:**

- Remove all comments from html
- Refactor all the repeated properties in CSS to ensure DRY

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**